Homework11/StrayPet.java

```diff
@@ -0,0 +1,45 @@
+package Homework11;
+
+import java.util.Scanner;
+
+@FunctionalInterface
+interface Taming {
+    HomePet tame(StrayPet strayPet, String newOwnerName);
+}
```

Comment:
наверное, я хотел что-то такое увидеть, но задача для этого не очень удачная, так что ок
```suggestion
interface Taming<T, R> {
    R tame(T strayPet, String newOwnerName);
}
```